driver/configuration/provider_viper_public_test.go

```diff
@@ -48,11 +48,9 @@ func TestPipelineConfig(t *testing.T) {
 		require.NoError(t, p.PipelineConfig("authenticators", "oauth2_introspection", nil, &res))
 		assert.JSONEq(t, `{"introspection_request_headers":{},"introspection_url":"https://override/path","pre_authorization":{"client_id":"some_id","client_secret":"some_secret","enabled":true,"scope":["foo","bar"],"token_url":"https://my-website.com/oauth2/token"},"required_scope":[],"scope_strategy":"exact","target_audience":[],"trusted_issuers":[]}`, string(res), "%s", res)
 
+		// Cleanup
 		require.NoError(t, os.Setenv("AUTHENTICATORS_OAUTH2_INTROSPECTION_CONFIG_INTROSPECTION_URL", ""))
 
-		require.NoError(t, p.PipelineConfig("authenticators", "oauth2_introspection", nil, &res))
```

Comment:
Why were these removed?

Reply:
This was basically the root cause of my intention to cache the environment ;) But as you've correctly pointed out there will be no changes to the env for the running process... So I've simply adjusted the test to reflect this.

Reply:
Ohh, I see - that totally makes sense :)